bitnami/cert-manager/values.yaml

```
@@ -567,7 +567,8 @@ webhook:
     ## @param webhook.serviceAccount.automountServiceAccountToken Automount service account token for the server service account
     ##
     automountServiceAccountToken: true
-
+  ##  @param webhook.hostNetwork Specifies hostNetwork value
```

Just to follow the same structure as the rest of parameters
```suggestion
  ##  @param webhook.hostNetwork Specifies hostNetwork value
  ##
```